src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -236,15 +236,17 @@ export function transform(formConfig, form) {
       claimedConditions,
     ).map(name => name.toLowerCase());
 
-    clonedData.newDisabilities = clonedData.newDisabilities.map(d => {
-      if (powDisabilities.includes(d.condition.toLowerCase())) {
-        const newSpecialIssues = (d.specialIssues || []).slice();
-        // TODO: Make a constant with all the possibilities and use it here
-        newSpecialIssues.push({ code: 'POW', name: '' });
-        return _.set('specialIssues', newSpecialIssues, d);
-      }
-      return d;
-    });
+    if (clonedData.newDisabilities) {
```

Comment:
Adding an undefined check since `clonedData.newDisabilities` is not always set in the `transform` function